src/main/resources/templates/add_sighting.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head lang="en"
+    xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    xmlns:th="https://www.thymeleaf.org">
+    <link th:href="@{/css/addSighting.css}" rel="stylesheet" />
+    <link th:href="@{/css/application.css}" rel="stylesheet" />
+    </head>
+    <body>
+        <div class="content-container">
+            <h1>Add a whale sighting</h1>
+            <p>Please add the details of your sighting below.dfrefgerg</p>
+            <form id="add-sighting" action="" method="">
```

Comment:
Please fill in `action` and `method` with the correct values

Reply:
Uncertain on the action for now - possibly need the th:action atrttribute instead? But have used # for now.